vmaas_sync/pkg_sync.go

```diff
@@ -5,12 +5,15 @@ import (
 	"app/base/database"
 	"app/base/models"
 	"app/base/utils"
+	"crypto/sha256"
 	"github.com/RedHatInsights/patchman-clients/vmaas"
 	"github.com/antihax/optional"
 	"github.com/jinzhu/gorm"
 	"github.com/pkg/errors"
 )
 
+const chunkSize = 10 * 1024
```

Comment:
I am not sure whether we use that convention everywhere, but I would prefer CHUNK_SIZE.

Reply:
It's a private constant.